apis/placement/v1/commons.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package v1
+
+const (
+	ClusterResourcePlacementKind        = "ClusterResourcePlacement"
+	ClusterResourcePlacementResource    = "clusterresourceplacements"
+	ClusterResourceBindingKind          = "ClusterResourceBinding"
+	ClusterResourceSnapshotKind         = "ClusterResourceSnapshot"
+	ClusterSchedulingPolicySnapshotKind = "ClusterSchedulingPolicySnapshot"
+	WorkKind                            = "Work"
+	AppliedWorkKind                     = "AppliedWork"
+)
+
+const (
+	// Unprefixed labels/annotations are reserved for end-users
+	// we will add a kubernetes-fleet.io to designate these labels/annotations as official fleet labels/annotations.
+	// See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions
+	fleetPrefix = "kubernetes-fleet.io/"
```

Comment:
need to think about putting them into a common package. otherwise, each version has its own consts

Reply:
I guess it kind of depends on how long we plan to keep the v1beta1 version; if not long I guess duplication would be OK?

Reply:
realistically, I think it will take at least 6 months